spec/integration/outputs/retry_spec.rb

```diff
@@ -30,8 +30,7 @@ def mock_actions_with_response(*resp)
       "manage_template" => true,
       "index" => "logstash-2014.11.17",
       "template_overwrite" => true,
-      "hosts" => get_host(),
-      "port" => get_port(),
+      "hosts" => get_host_port(),
```

Comment:
Oops, responded to the wrong thing. Hmmm, that is just historical. Does the style guide say anything here? (I personally prefer ())